pyproject.toml

```diff
@@ -37,6 +35,9 @@ ipdb = { version = ">=0.13.13", optional = true }
 jupyter = { version = ">=1.0.0", optional = true }
 pytest = { version = ">=7.4.2", optional = true }
 pytest-repeat = { version = ">=0.9.1", optional = true }
+gflownet = { git = "https://github.com/alexhernandezgarcia/gflownet-dev", branch = "catalyst", optional = true }
+# gflownet = { git = "https://github.com/alexhernandezgarcia/gflownet", branch = "activelearning", optional = true }
```

Comment:
Are you sure this is correct?

Reply:
it didn't work when calling it gflownetdev. maybe because internally the project is called gflownet?

or what do you mean?